tests/nativeapp/codegen/snowpark/__snapshots__/test_python_processor.ambr

```diff
@@ -545,8 +545,8 @@
 # name: test_process_with_collected_functions.1
```

Comment:
the iteration order happened to get reversed when adding `sorted()`, so the data in the snapshot here is reversed